slides/src/content/component.md

```diff
@@ -112,13 +112,26 @@ _src/app/app.component.html_
 
 - Whoops: compiler error!
 
-_src/app/app.component.html_ (with error)
+_src/app/app.component.html_
 ```html
 <app-to-do-list [heading]="'Things To Do'"></app-to-do-list>
 ```
 
 - Have to quote the value inside the double quotes or Angular will try to evaluate it
 
+
+---
```

Comment:
Uh oh - this will introduce a new slide and throw off our slide numbering, yes?  We'll discuss at 2017-04-21's retro what to do about identifying slides when this happens.